README.md

```diff
@@ -10,34 +10,36 @@ POST /1/login
   * Body: {user: `<uuid>`, password: "secret"}
   * Returns: 200 + a cookie
 
-GET /1/games
+GET /1/opengames
   * Get a list of available games to join
-  * Returns: {games: [{game: `<uuid>`, opponent: `<uuid>`, name: "death match"}, ...]}
-
-GET /1/game/start?join=`<bool>`&user=`<uuid>`
-  * Initialize a new game from an anonymous user
-  * Optional query param "join" allows instant joining of any available game
-  * Optional query param "user" allows joining as a know user
-  * Returns: {game: `<uuid>`, user: `<uuid>`, status: "waiting"|"started", [state: `<boardState>`, color: "white"|"black"]}
-  * A cookie is also set if a game is anonymously joined
+  * Returns: {games: [{game: `<uuid>`, opponent: `<uuid>`}, ...]}
+
+GET /1/mygames?user=`<uuid>`
```

Comment:
If we are using cookies, then we don't strictly need user in the parameter.
No harm in having it, just saying